editor/src/core/model/project-file-utils.ts

```diff
@@ -164,6 +165,13 @@ function isHTMLComponentFromBaseName(baseName: string, imports: Imports): boolea
   }
 }
 
+export function isImportedComponent(jsxElementName: JSXElementName, imports: Imports): boolean {
+  return Object.keys(imports).some((importKey) => {
```

Comment:
I don't believe this covers all the cases for how something may be imported, like if a component is the default export.